docs/draft/rest_api.md

```diff
@@ -0,0 +1,34 @@
+# First Draft for the REST API
```

Comment:
This comment is about the whole Document. This might because of it beeing a very eary draft. What do the API Calls return? JSON?